templates/taxi/index.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% block content %}
+  <ul>
+  <li>Number of cars: {{ num_cars }}</li>
+  <li>Number of drivers: {{ num_driver }}</li>
```

Comment:
There is a typo in the variable name `num_driver`. It should be `num_drivers` to match the context dictionary key defined in the `views.py` file.